example/explicit_module/BUILD.bazel

```diff
@@ -1,23 +1,12 @@
-load("@build_bazel_rules_swift//swift:swift.bzl", "swift_binary")
-load("//rules/explicit_module:sdk_clang_module.bzl", "sdk_clang_module")
-
-sdk_clang_module(
-    name = "swift_concurrency_shims",
-    module_map = "__BAZEL_XCODE_SDKROOT__/usr/lib/swift/shims/module.modulemap",
-    module_name = "_SwiftConcurrencyShims",
-)
-
-sdk_clang_module(
-    name = "shims",
-    module_map = "__BAZEL_XCODE_SDKROOT__/usr/lib/swift/shims/module.modulemap",
-    module_name = "SwiftShims",
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_binary",
 )
 
 swift_binary(
     name = "hello_world",
     srcs = ["main.swift"],
     deps = [
-        ":shims",
-        ":swift_concurrency_shims",
+        "@xcode_sdk_frameworks//version15_4_0_15F31d/MacOSX:Foundation_swift",
```

Comment:
possible to have this as a separate var that can be specified next to `build:explicit_module --xcode_version=xxx`? this way we at least control the xcode version EMB points to easily